src/SpecFlow.NetCore/AppConfig.cs

```diff
@@ -52,16 +54,34 @@ public static AppConfig CreateIn(DirectoryInfo directory)
 			return config;
 		}
 
-		public void Validate()
+		private static string GuessUnitTestProvider(DirectoryInfo directory)
 		{
-			WriteLine("Validating app.config.");
-
-			// I would rather use the ConfigurationBuilder, but as of beta8 it fails to read an element without
-			// a value, e.g.: <unitTestProvider name="xUnit" />
-			//var config = new ConfigurationBuilder()
-			//	.AddXmlFile(path)
-			//	.Build();
+			WriteLine("Figuring out test runner. You can always change this later in app.config.");
+			var foundProjectJson = directory.GetFiles().SingleOrDefault(f => f.Name == "project.json");
+			if (foundProjectJson != null)
+			{
+				dynamic projectJson = JsonConvert.DeserializeObject(File.ReadAllText(foundProjectJson.FullName));
+				if (projectJson.testRunner != null)
```

Comment:
Just realised dynamics don't work like this and this will likely throw. Will fix tomorrow